lib/utils.js

```diff
@@ -158,6 +160,47 @@ exports.trimUnderscores = function(name) {
 };
 
 /**
+ * Whether or not the given path is relative
+ * @param  {String}  path
+ * @return {Boolean}
+ */
+exports.isRelativePath = function(path) {
+    // Logic from: https://github.com/joyent/node/blob/4f1ae11a62b97052bc83756f8cb8700cc1f61661/lib/module.js#L237
+    var start = path.substring(0, 2);
+    return start === './' || start === '..';
+};
+
+/**
+ * Shallow extend
+ * @param  {Object} obj1 - Object to be extended
+ * @param  {Object} obj2
+ * @return {Object} A reference to obj1
+ */
+exports.extend = function(obj1, obj2) {
+    Object.keys(obj2).forEach(function(prop) {
+        obj1[prop] = obj2[prop];
+    });
+
+    return obj1;
+};
+
+/**
+ * Resolves a relative filepath against the supplied base path
+ * or just returns the filepath if not relative
+ *
+ * @param  {String} filepath
+ * @param  {String} basePath
+ * @return {String}
+ */
+exports.normalizePath = function(filepath, basePath) {
```

Comment:
It's used in generator and in configuration. The only suitable location aside from utils would be `checker` if we want to avoid a PathHelper.